privacy-config/privacy-config-store/src/main/java/com/duckduckgo/privacy/config/store/PrivacyConfigDatabase.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2021 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.privacy.config.store
+
+import android.content.Context
+import androidx.room.Database
+import androidx.room.Room
+import androidx.room.RoomDatabase
+
+@Database(
+    exportSchema = true, version = 1,
+    entities = [
+        ContentBlockingException::class,
+        PrivacyFeatureToggles::class
+    ]
+)
+abstract class PrivacyConfigDatabase : RoomDatabase() {
+
+    abstract fun contentBlockingDao(): ContentBlockingDao
+    abstract fun privacyFeatureTogglesDao(): PrivacyFeatureTogglesDao
+
+    companion object {
+        @Volatile
+        private var INSTANCE: PrivacyConfigDatabase? = null
+
+        fun getInstance(context: Context): PrivacyConfigDatabase =
+            INSTANCE ?: synchronized(this) {
+                INSTANCE ?: buildDatabase(context).also { INSTANCE = it }
+            }
```

Comment:
I wonder if, given that the `providePrivacyConfigDatabase` is marked as `Singleton` already, we can probably remove all the ceremony to create a singleton manually (?)

Reply:
Makes sense, changed!